.classpath

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java">
+		<attributes>
```

Comment:
Shall we remove this .xxxx files. Do we need these to do a maven clean repo build without an IDE